app/controllers/connections_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ConnectionsController < ApplicationController
+  before_action :require_login
+
+  def index
+    @connections = current_user.following_connections
+  end
+
+  def destroy
+    @connection = Connection.find params[:id]
+    @connection.delete if @connection.follower == current_user
```

Comment:
Since you redirect, there isn't a need to use instance variables here. You won't keep the instance and template won't be rendered for this action.